build.gradle

```diff
@@ -953,13 +953,15 @@ project(':clients') {
     compile libs.lz4
     compile libs.snappy
     compile libs.slf4jApi
-
+    compile libs.shiro
```

Comment:
Had a deeper look into this Shiro library. On one hand it supports many things, like remember-me, caching, authorization, sessions, etc.
On the other hand I'm not sure if we need all this functionality (like caching or sessions) as Kafka either doesn't require all this (caching, rememberme) or implements some things on its own (sessions). Therefore it might be better to implement the required LDAP functionality with Directory itself.

Reply:
At the moment I am not sure what is best to use. In the code I see apache directory api being used only for test at the moment (`testCompile(libs.apacheda)`), so either way, either using Apache Directory or Apache Shiro will add additional dependency to the clients project. Also, Shiro can be utilized for future improvement such as supporting Linux Pluggable Authentication Modules (PAM), so clients with LDAP PAM setup will can authenticate seamlessly.